src/Readline/Readline.php

```diff
@@ -80,4 +82,14 @@ public function redisplay();
      * @return bool Success
      */
     public function writeHistory(): bool;
+
+    /**
+     * Activete auto completer for tab completion.
+     */
+    public function activateAutoCompleter(AutoCompleter $autoCompleter): void;
```

Comment:
let's make these two a second interface so we can maintain backwards compatibility for existing Readline API.